transit_odp/fares_validator/views/validate.py

```diff
@@ -48,6 +52,10 @@ def set_errors(self):
             response = JsonResponse(
                 serializer.data, safe=False, status=status.HTTP_201_CREATED
             )
+        # For 'Update data' flow
+        FaresValidationResult.objects.filter(
+            revision_id=self.pk2, organisation_id=self.pk1
+        ).delete()
         FaresValidationResult.create_validation_result(
             revision_id=self.pk2, org_id=self.pk1, violations=violations
         ).save()
```

Comment:
Please add `with transaction.atomic():` for all or nothing for this entire deletion and addition of records for both the tables i.e from line 41 to 61..

Reply:
Done this change